lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -144,6 +155,45 @@ bool PatchInOutImportExport::runOnModule(Module &module) {
   return true;
 }
 
+// =====================================================================================================================
+// Mark the 'done' flag to the very last position export instruction.
+//
+// @param [in/out] func : LLVM function to be run on
+// @param [in] pdt : The PostDominatorTree of the \p func
+void PatchInOutImportExport::markExportDone(Function *func, PostDominatorTree &pdt) {
+  std::vector<CallInst *> expInstrs;
+
+  Function *expDecl = m_module->getFunction("llvm.amdgcn.exp.f32");
+  if (!expDecl)
+    return;
+
+  // get the export call instructions
```

Comment:
nit: capitalize `get'

Reply:
Is the order of collected exports arbitrary here? Later, you use first export in `lastExport`.

Reply:
I would assume the users() are in random order. I am just trying to find the export that post-dominates the other exports in `expInstrs` and mark the done flag.

Reply:
Ack.